src/Client.php

```diff
@@ -202,8 +202,9 @@ public function createRequest()
      * @return Response request instance.
      * @throws \yii\base\InvalidConfigException
      */
-    public function createResponse($content = null, array $headers = [])
+    public function createResponse($content = null, $headers = null)
```

Comment:
This is not BC, since it changes method signature. What if we use:

```suggestion
    public function createResponse($content = null, array $headers = null)
```

Reply:
@rob006, I think it's completely backward compatible. Could you think of any case that can break after this change?

Reply:
If I'm already extending this method with old signature, I will get warning after this change: https://3v4l.org/3fTW0

Also removing `array` typehint is just less safe, so I'm not sure what we're win by this.

Reply:
Using `array $headers = null` will also trigger warning in this case.
We can move a casting to an array to StreamTransport before passing header to createResponse, but this issue can occur in other transports as well.
So, any ideas which case is better?

Reply:
Yes. Changing signature isn't what we can merge into 2.x.

Reply:
@rob006 original change: https://3v4l.org/gsuIm

Reply:
Made changes.

Reply:
@rob006 your proposal triggers a warning as well: https://3v4l.org/YhqPu